app/models/comment.rb

```diff
@@ -1,4 +1,8 @@
 class Comment < ApplicationRecord
-  belongs_to :user
-  belongs_to :post
+  belongs_to :author, foreign_key: :author_id, class_name: 'User'
+  belongs_to :post, class_name: 'Post', counter_cache: :comments_counter
+
+  def update_post_comment_counter
+    post.increment!(:comments_counter)
+  end
```

Comment:
- [ ] [OPTIONAL] Please use the `after_save` rails callback here to update the post's comments counter just like I suggested for `posts`

Reply:
Thanks. So, cool